docs/guide/tiagopp.md

```diff
@@ -1,47 +1,52 @@
-## PAL Robotics' TIAGopp
+## PAL Robotics' TIAGo ++
```

Comment:
It seems the correct spelling has no space before ++, see https://pal-robotics.com/robots/tiago/
```suggestion
## PAL Robotics' TIAGo++
```